src/ResourceManager/Cdn/Commands.Cdn/Endpoint/NewAzureRmCdnEndpoint.cs

```diff
@@ -90,9 +90,15 @@ public class NewAzureRmCdnEndpoint : AzureCdnCmdletBase
         [Parameter(Mandatory = false, HelpMessage = "Specifies any optimization this endpoint has.")]
         public string OptimizationType { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Specifies the probe path for Dynamic Site Acceleration")]
+        public string ProbePath { get; set; }
+
         [Parameter(Mandatory = false, HelpMessage = "The list of geo filters that applies to this endpoint.")]
         public PSGeoFilter[] GeoFilters { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "The delivery policy for this endpoint.")]
+        public PSDeliveryPolicy DeliveryPolicy { get; set; }
```

Comment:
@prakharsharma10 how do users create this object?

Reply:
Delivery Policy is not a resource, it's a property. The users can create this object as shown in the EndpointTests.ps1, the same way they create geofilters